llvm/docs/LangRef.rst

```diff
@@ -11720,9 +11730,8 @@ Note that ``getelementptr`` with all-zero indices is always considered to be
 As a corollary, the only pointer in bounds of the null pointer in the default
 address space is the null pointer itself.
 
-These rules are based on the assumption that no allocated object may cross
-the unsigned address space boundary, and no allocated object may be larger
-than half the pointer index type space.
+These rules are based on the assumption for
```

Comment:
```suggestion
These rules are based on the assumptions for
```
Though I'm not sure we really need this sentence anymore, it may make more sense to link "allocated object" in the bullet point above.

Reply:
Updated the bullet point to link to the allocated objects section and remove the sentence here.